### ISSUE TITLE
Не устанавливается на Laravel 7.4

### ISSUE THREAD

**Issue by Inogami** [vyuldashev/laravel-queue-rabbitmq#329]
- Laravel Version: 7.4.0
- Package Version: 10.1.3
- RabbitMQ Version: 3.8.3

Не устанавливается зависимость

```
  Problem 1
    - Conclusion: remove laravel/framework v7.4.0
    - Conclusion: don't install laravel/framework v7.4.0
    - vladimir-yuldashev/laravel-queue-rabbitmq v10.1.0 requires illuminate/queue ^6.0 -> satisfiable by illuminate/queue[6.x-dev, v6.0.0, v6.0.1, v6.0.2, v6.0.3, v6.0.4, v6.1.0, v6.10.0, v6.11.0, v6.12.0, v6.13.0, v6.13.1, v6.14.0, v6.15.0, v6.15.1, v6.16.0, v6.17
.0, v6.17.1, v6.18.0, v6.18.1, v6.18.2, v6.18.3, v6.2.0, v6.3.0, v6.4.1, v6.5.0, v6.5.1, v6.5.2, v6.6.0, v6.6.1, v6.6.2, v6.7.0, v6.8.0].
    - vladimir-yuldashev/laravel-queue-rabbitmq v10.1.1 requires illuminate/queue ^6.0 -> satisfiable by illuminate/queue[6.x-dev, v6.0.0, v6.0.1, v6.0.2, v6.0.3, v6.0.4, v6.1.0, v6.10.0, v6.11.0, v6.12.0, v6.13.0, v6.13.1, v6.14.0, v6.15.0, v6.15.1, v6.16.0, v6.17
.0, v6.17.1, v6.18.0, v6.18.1, v6.18.2, v6.18.3, v6.2.0, v6.3.0, v6.4.1, v6.5.0, v6.5.1, v6.5.2, v6.6.0, v6.6.1, v6.6.2, v6.7.0, v6.8.0].
    - vladimir-yuldashev/laravel-queue-rabbitmq v10.1.2 requires illuminate/queue ^6.0 -> satisfiable by illuminate/queue[6.x-dev, v6.0.0, v6.0.1, v6.0.2, v6.0.3, v6.0.4, v6.1.0, v6.10.0, v6.11.0, v6.12.0, v6.13.0, v6.13.1, v6.14.0, v6.15.0, v6.15.1, v6.16.0, v6.17
.0, v6.17.1, v6.18.0, v6.18.1, v6.18.2, v6.18.3, v6.2.0, v6.3.0, v6.4.1, v6.5.0, v6.5.1, v6.5.2, v6.6.0, v6.6.1, v6.6.2, v6.7.0, v6.8.0].
    - vladimir-yuldashev/laravel-queue-rabbitmq v10.1.3 requires illuminate/queue ^6.0 -> satisfiable by illuminate/queue[6.x-dev, v6.0.0, v6.0.1, v6.0.2, v6.0.3, v6.0.4, v6.1.0, v6.10.0, v6.11.0, v6.12.0, v6.13.0, v6.13.1, v6.14.0, v6.15.0, v6.15.1, v6.16.0, v6.17
.0, v6.17.1, v6.18.0, v6.18.1, v6.18.2, v6.18.3, v6.2.0, v6.3.0, v6.4.1, v6.5.0, v6.5.1, v6.5.2, v6.6.0, v6.6.1, v6.6.2, v6.7.0, v6.8.0].
    - don't install illuminate/queue 6.x-dev|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.0.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.0.1|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.0.2|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.0.3|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.0.4|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.1.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.10.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.11.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.12.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.13.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.13.1|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.14.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.15.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.15.1|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.16.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.17.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.17.1|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.18.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.18.1|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.18.2|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.18.3|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.2.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.3.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.4.1|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.5.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.5.1|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.5.2|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.6.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.6.1|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.6.2|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.7.0|don't install laravel/framework v7.4.0
    - don't install illuminate/queue v6.8.0|don't install laravel/framework v7.4.0
    - Installation request for laravel/framework 7.4.0 -> satisfiable by laravel/framework[v7.4.0].
    - Installation request for vladimir-yuldashev/laravel-queue-rabbitmq ^10.1 -> satisfiable by vladimir-yuldashev/laravel-queue-rabbitmq[v10.1.0, v10.1.1, v10.1.2, v10.1.3].

```

**Comment by rdcx** [comment]
@vladkucherov is this something that can be easily fixed or will it take some changes?

**Comment by Norgul** [comment]
I have updated dependencies on my repo `https://github.com/Norgul/laravel-queue-rabbitmq`.
You can include it like:

```
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/norgul/laravel-queue-rabbitmq"
        }
    ],
    "require": {
        "vladimir-yuldashev/laravel-queue-rabbitmq": "dev-master"
    },
```

**Comment by uncle-tee** [comment]
@Norgul  Thanks it works But please can you please request for PR. Thanks.

**Comment by Norgul** [comment]
Sure, submitted.

**Comment by vyuldashev** [comment]
v10.2.0 released with support for Laravel 7.0
